index.html

```diff
@@ -65,7 +65,7 @@
 			<span class="title">about</span>
 			<div class="columns group with-button">
 				<div class="col-xs-12 col-md-8 col-md-offset-2 content">
-					EmpowerHack is a collective that brings creates sustainable design for women and girls in humanitarian contexts. Bridging the gap between technology, design, and NGOs to address key challenges, Empower Hack galvanises volunteers to create code that crosses borders.<br /><br />
+					EmpowerHack is a collective that creates sustainable design for women and girls in humanitarian contexts. Bridging the gap between technology, design, and NGOs to address key challenges, EmpowerHack galvanises volunteers to create code that crosses borders.<br /><br />
```

Comment:
Awesome! Yes I agree, do it as a separate **Pull Request**. We want **Pull Requests** small and short lived 😄 . I am keen to get this merged today 👍